WLC/benchmark.py

```diff
@@ -45,20 +48,25 @@ def run_benchmarks():
     LOGGER.info('')
 
     total_accuracy = 0
+    total_accuracy_fixed = 0
     total_length = 0
 
     for i in range(1, 9):
-        accuracy, length = benchmark_file('assets/examples/images/example_{}.png'.format(i))
+        accuracy, accuracy_fixed, length = benchmark_file('assets/examples/images/example_{}.png'.format(i))
         total_accuracy += accuracy * length
+        total_accuracy_fixed += accuracy_fixed * length
         total_length += length
 
     overall_accuracy = round(total_accuracy / total_length)
+    overall_accuracy_fixed = round(total_accuracy_fixed / total_length)
 
     LOGGER.info('')
-    LOGGER.info('Overall Accuracy: {}%'.format(overall_accuracy))
+    LOGGER.info('Overall Accuracy w/o fix: {}%'.format(overall_accuracy))
+    LOGGER.info('Overall Accuracy w/ fix: {}%'.format(overall_accuracy_fixed))
+    LOGGER.info('Fix improvement: {}%'.format(overall_accuracy_fixed - overall_accuracy))
```

Comment:
Technically should use %s instead of format in logging calls (and parse args as args to the function), but this is a tiny optimisation for when a statement isnt logged, which they all are at the moment